zeppelin-plugins/launcher/yarn/src/test/java/org/apache/zeppelin/interpreter/launcher/YarnLauncherUtilTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zeppelin.interpreter.launcher;
+
+import org.junit.Test;
+
+import java.net.URI;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class YarnLauncherUtilTest {
+
+  @Test
+  public void testURIUtil() {
+    URI uri = YarnLauncherUtil.resolveURI("/tmp/env_1");
+    assertEquals("file", uri.getScheme());
+    assertEquals("/tmp/env_1", uri.getPath());
+    assertNull(uri.getFragment());
+
+    uri = YarnLauncherUtil.resolveURI("/tmp/env_1#env");
```

Comment:
Thank you for the tests. I think we should add some more tests.
1) a string with a scheme
2) a relative path

Reply:
Fixed